web-app/css/general.css

```diff
@@ -1118,3 +1117,8 @@ div.tool-plus-disabled {
     opacity: 0.8;
     font-family: FontAwesome! important;
 }
+
+.fa-lg.fa-angle-double-right,
```

Comment:
All the rest of the times you have both in the same line. I don't think it's critical, it's just inconsistent.